yarn.lock

```diff
@@ -3470,13 +3470,35 @@
     timsort "~0.3.0"
     z-schema "~3.18.3"
 
-"@rushstack/package-deps-hash@^2.4.109", "@rushstack/package-deps-hash@^2.4.48":
+"@rushstack/node-core-library@3.45.1":
+  version "3.45.1"
+  resolved "https://registry.yarnpkg.com/@rushstack/node-core-library/-/node-core-library-3.45.1.tgz#787361b61a48d616eb4b059641721a3dc138f001"
+  integrity sha512-BwdssTNe007DNjDBxJgInHg8ePytIPyT0La7ZZSQZF9+rSkT42AygXPGvbGsyFfEntjr4X37zZSJI7yGzL16cQ==
+  dependencies:
+    "@types/node" "12.20.24"
+    colors "~1.2.1"
+    fs-extra "~7.0.1"
+    import-lazy "~4.0.0"
+    jju "~1.4.0"
+    resolve "~1.17.0"
+    semver "~7.3.0"
+    timsort "~0.3.0"
+    z-schema "~5.0.2"
+
+"@rushstack/package-deps-hash@^2.4.48":
```

Comment:
what is causing 2 of the same rushtack package-deps-hash packages to be installed ?

Reply:
^ I have same question :D. I hope we can get rid of this 🤞

Reply:
rushtack package-deps hash version `2.4.48` was always there but version `3.2.5` was brought in after `just-scripts` was upgraded to version `1.8.2` since one of its dependencies (`just-task`) has a dependency on that rush version 😬